tutorials/spa-create-excel-automation/spa-create-excel-automation.md

```diff
@@ -0,0 +1,199 @@
+---
+author_name: Ramakrishnan Raghuraman
+author_profile: https://github.com/r3ksk
+title: Build your first automation using Excel SDK of SAP Process Automation
+description: In this tutorial, you will be developing basic automations using Excel SDK, In the end, you will know how to read excel data for a given range using **Excel Cloud Link**, or using traditional approach. In this tutorial, we will show how you can apply filter on the excel data, copy the filtered data onto a new excel sheet and save the file with a new name.
+auto_validation: true
+time: 30
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform, tutorial>free-tier]
+primary_tag: software-product>sap-process-automation
+---
+
+## Details
+
+SAP Process Automation has native integration to several Microsoft Office products including Outlook, Excel SDK. In this Tutorial, you will explore how to use Excel SDK to your automations.
+
+## Prerequisites
+
+- Complete the tutorial: [Subscribe to SAP Process Automation Using Booster in SAP BTP Free Tier](spa-subscribe-booster)
+- Complete the tutorial: [Install and Set Up the Desktop Agent](spa-setup-desktop-agent)
+- Familiarity to create an automation using SAP Process Automation
+
+### You will learn
+
+  - How to use Excel SDK of SAP Process Automation (SPA) to read and process data in an excel
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](About Excel SDK)]
+
+The Excel SDK is a collection of activities allowing you to create automations using Microsoft Excel.
+It comes with rich set of functionality. For ex,
+
+- Create new excel Workbook
+- Create new excel Worksheet
+- Read and write data from an excel sheet, for a given data range
+- Apply Filter on the a column
+- etc
```

Comment:
I would not use "etc"  If there are more functions. list them, otherwise end the list.